docs/user_guide.rst

```diff
@@ -170,16 +170,73 @@ Now we can study how the circuit gets transpiled:
+real hardware, by executing it on an :class:`.IQMFakeBackend` instance: 
 
 .. code-block:: python
 
-    from qiskit import Aer
+    from qiskit import execute, QuantumCircuit
+    from qiskit_iqm import IQMFakeAdonis
+    from qiskit.compiler import transpile
+
+    circuit = QuantumCircuit(2)
+    circuit.h(0)
+    circuit.cx(0, 1)
+    circuit.measure_all()
+
+    backend = IQMFakeAdonis()
+    circuit = transpile(circuit, backend=backend)
+    job = execute(circuit, backend, qubit_mapping=None, shots=1000)
+    job.result().get_counts()
+
+
+Above we use an :class:`.IQMFakeAdonis` instance to run a noisy simulation of ``circuit``
+after transpiling it for that specific backend.
+
+:class:`IQMFakeBackend` can be initialized using an :class:`IQMChipSample` instance, which describes the
+physical properties of the QPU specimen to be simulated. If not provided, by default each
+:class:`IQMFakeBackend` subclass uses representative values for that QPU type.
+
+If you want to use your own noise parameters, you can create your own chip sample as follows:
+
+.. code-block:: python
+
+    from qiskit_iqm import IQMChipSample
+    from iqm_client import QuantumArchitectureSpecification
+    from qiskit_iqm import IQMFakeBackend
+
+    some_chip_sample = IQMChipSample(
+        quantum_architecture=QuantumArchitectureSpecification(
+            name="Adonis",
+            operations=["phased_rx", "cz", "measurement", "barrier"],
+            qubits=["QB1", "QB2", "QB3", "QB4", "QB5"],
+            qubit_connectivity=[["QB1", "QB3"], ["QB2", "QB3"], ["QB3", "QB4"], ["QB3", "QB5"]],
+        ),
+        # T1 times in ns for all qubits, index indicates the qubit number
+        t1s={"QB1": 50000.0, "QB2": 50000.0, "QB3": 50000.0, "QB4": 50000.0, "QB5": 50000.0},
+        # T2 times in ns for all qubits, index indicates the qubit number
+        t2s={"QB1": 50000.0, "QB2": 50000.0, "QB3": 50000.0, "QB4": 50000.0, "QB5": 50000.0},
+        # Gate fidelities for one and two qubit gates, the key in the dictionary indicates the qubits the gate operates on
+        one_qubit_gate_fidelities={"r": {"QB1": 0.999, "QB2": 0.999, "QB3": 0.999, "QB4": 0.999, "QB5": 0.999}},
+        two_qubit_gate_fidelities={"cz": {("QB1", "QB3"): 0.999, ("QB2", "QB3"): 0.999, ("QB4", "QB3"): 0.999, ("QB5", "QB3"): 0.999}},
+        # Gate depolarization rates for one and two qubit gates, the key in the dictionary indicates the qubits the gate operates on
+        one_qubit_gate_depolarization_rates={"r": {0: 0.0001, 1: 0.0001, 2: 0.0001, 3: 0.0001, 4: 0.0001}},
+        two_qubit_gate_depolarization_rates={"cz": {("QB1", "QB3"): 0.001, ("QB2", "QB3"): 0.001, ("QB4", "QB3"): 0.001, ("QB5", "QB3"): 0.001}},
+        # Gate durations in ns
+        one_qubit_gate_durations={"r": 40.0},
+        two_qubit_gate_durations={"cz": 80.0},
+        id_="sample-chip",
+    )
+
+    IQMFakeBackend(some_chip_sample)
```

Comment:
I think this part of the user guide is to too verbose. We could just point to the documentation of chip sample which already contains an example of creating a chip sample instance, and IQMFakeBackend documentation contains example how to create fake backend